recipes-bsp/u-boot/u-boot-imx_2017.03.bb

```diff
@@ -20,5 +20,29 @@ inherit dtc-145 fsl-u-boot-localversion
 
 LOCALVERSION ?= "-${SRCBRANCH}"
 
+BOOT_TOOLS = "imx-boot-tools"
+
+do_deploy_append_mx8mq () {
+    # Deploy the mkimage, u-boot-nodtb.bin and fsl-imx8mq-XX.dtb for mkimage to generate boot binary
+    if [ -n "${UBOOT_CONFIG}" ]
+    then
+        for config in ${UBOOT_MACHINE}; do
+            i=$(expr $i + 1);
+            for type in ${UBOOT_CONFIG}; do
+                j=$(expr $j + 1);
+                if [ $j -eq $i ]
+                then
+                    install -d ${DEPLOYDIR}/${BOOT_TOOLS}
+                    install -m 0777 ${B}/${config}/arch/arm/dts/${UBOOT_DTB_NAME}  ${DEPLOYDIR}/${BOOT_TOOLS}
+                    install -m 0777 ${B}/${config}/tools/mkimage  ${DEPLOYDIR}/${BOOT_TOOLS}/mkimage_uboot
```

Comment:
This should be moved to a `u-boot-imx-mx8boot` or similar, so it can be built as part of native and also installed in SDK.